covid-vaccine-faq-bot/functions/auth.private.js

```diff
@@ -0,0 +1,78 @@
+async function getCurrentEnvironment(context) {
```

Comment:
this file seems very similar to that of the one in the vaccine-standby app we already have merged in. I would use that as a very close reference since that file is formatted to fix all the comments ive listed down below